lib/Slider/Slider.js

```diff
@@ -269,19 +269,14 @@ module.exports = kind(
 	* @private
 	*/
 	jumpWrapperComponents: [
-		{name: 'buttonLeft', kind: IconButton, backgroundOpacity: 'transparent', classes: 'moon-slider-button left', icon: 'arrowlargeleft', onSpotlightKeyDown: 'configureSpotlightHoldPulse', onSpotlightSelect: 'previous', ondown: 'downLeft', onholdpulse: 'holdLeft', ondragstart: 'preventDrag', defaultSpotlightDisappear: 'buttonRight'},
+		{name: 'buttonLeft', kind: IconButton, backgroundOpacity: 'transparent', classes: 'moon-slider-button left', icon: 'arrowlargeleft', onSpotlightKeyDown: 'showKnobStatus', onSpotlightKeyUp: 'hideKnobStatus', onSpotlightSelect: 'previous', ondown: 'downLeft', onup: 'hideKnobStatus', onholdpulse: 'holdLeft', ondragstart: 'preventDrag', defaultSpotlightDisappear: 'buttonRight'},
```

Comment:
Ah ha I see. :smile: I think I needed to tweak one thing with my branch - with the way it is implemented, there would be no need to `return true` from the `next` and `previous` methods. Let me know how it works out for you.